src/App.scss

```diff
@@ -1,37 +1,65 @@
 // Importing Global Variables
-@import '~@red-hat-insights/insights-frontend-components/Utilities/_all';
-@import '~@red-hat-insights/insights-frontend-components/index.css';
-@import '~@red-hat-insights/insights-frontend-components/Utilities/_mixins';
+// Sass Imports
+@import "~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_mixins";
+@import "~@redhat-cloud-services/frontend-components-utilities/files/Utilities/all";
 
-.pf-l-page__main .pf-c-card { min-height: 100%; }
+@import "~@redhat-cloud-services/frontend-components/index.css";
+
+.pf-l-page__main .pf-c-card {
+    min-height: 100%;
+}
 
 .ins-c-summary {
-    &__accent { color: var(--pf-global--palette--black-500); }
+    &__accent {
+        color: var(--pf-global--palette--black-500);
+    }
 
-    &__emphasis { @include rem("font-size", 32px); }
+    &__emphasis {
+        @include rem("font-size", 32px);
+    }
 
     &__icon {
```

Comment:
I might be wrong, but I don't know if we use all of these colors?

Reply:
Ooooo no idea, not keen on touching that, this is just auto linting cuz the file was touched

Reply:
ahhh ok, all good then!